lib/bamboo/mailer.ex

```diff
@@ -184,8 +184,7 @@ defmodule Bamboo.Mailer do
   end
 
   @doc false
-  def parse_opts(mailer, opts) do
-    otp_app = Keyword.fetch!(opts, :otp_app)
+  def parse_opts(mailer, otp_app) do
```

Comment:
I think it's ok because it's not documented and probably shouldn't be used by other devs anyway :)